### DISCUSSION TITLE
Cursor not updating if the mouse is not moving

### DISCUSSION THREAD

**Discussion by KreNtal** [pixijs/pixijs#9561] [Ideas]
As said in the title the cursor is not updating if the mouse is not moving, that was working as expected in version 6.

**Comment by gabrielliwerant** [comment]
This is unfortunate. Just curious, but as a workaround, have you tried programmatically changing the cursor position by a pixel to induce the cursor update? Wondering if that works. It's not ideal, but shifting the cursor back and forth by a pixel might not be too noticeable.

**Comment by mklfarha** [comment]
setting the eventMode to 'dynamic' on the container fixed the issue for me (version: 8.3.2)

**Reply by FelixNumworks** [reply]
Thank you this is exactly what I needed
